notebooks/benchmark.ipynb

```diff
@@ -85,7 +85,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# my_data = b.make_dataset(\"/PATH/TO/MY_DATASET.csv\", name=\"my-data\", datatype=\"INDICATE_DATATYPE\")"
+    "# my_data = b.create_dataset(\"/PATH/TO/MY_DATASET.csv\", name=\"my-data\", datatype=\"INDICATE_DATATYPE\")"
```

Comment:
Is it worth calling out here (or in the docs) that `make_dataset` has been deprecated?

Reply:
I'll definitely include it in the Trainer release notes when it we publish v2 ([examples](https://github.com/gretelai/trainer/releases)). Personally I would have the docs just present valid/current stuff and not mention in there (more confusing than good for new users), but I'll let @hhpang make the final call on that. (We will have some updates to make to the docs regardless.)

Reply:
oh man i'm so sorry i missed this message somehow. looks like we didn't call it out here, sgtm